code/modules/mob/living/say.dm

```diff
@@ -70,7 +70,7 @@ proc/get_radio_key_from_channel(var/channel)
 /mob/living/proc/handle_speech_problems(var/message, var/verb)
 	var/list/returns[3]
 	var/speech_problem_flag = 0
-
+	var/robot = isSynthetic()

```

Comment:
some things, you can.

For example, reagent datums have a color, which you can defined as a hex string text OR `rgb(0,0,0)`